webauthn/src/WebAuthn.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
```

Comment:
Apache-2.0 is an unusual choice for us. Did this code come from somewhere?

Reply:
Got it from somewhere else. Changed file and changed license to MIT